desktop_native/src/password/unix.rs

```diff
@@ -0,0 +1,83 @@
+use anyhow::{anyhow, Result};
+use libsecret::{password_clear_sync, password_lookup_sync, password_store_sync, Schema};
```

Comment:
Any chance we can use the `future` versions of these and actually put the `async` to use?

Reply:
The future apis are a bit complicated as they expect a main gio thread + a  loop. I should probably investigate this a bit further as some users have reported issues with the linux app locking when libsecrets can't communicate.